mitodl/mitxonline
# Update mitol-django-google-sheets

### What are the relevant tickets?
Related to https://github.com/mitodl/ol-django/pull/159

### Description (What does it do?)
Update mitol-django-google-sheets to 2024.7.3

### How can this be tested?
You need a Google folder set up to be processed by your local mitxonline instance.

Make sure the sheet is being processed, and setting MITOL_GOOGLE_SHEETS_PROCESS_ONLY_LAST_ROWS_NUM has an effect on how many rows get processed.
